EmojiTools/EmojiTools.swift

```diff
@@ -40,6 +36,22 @@ extension String {
         return characters
     }
     
+    /**
+     Converts emoji markdown to an emoji. For example, ":smile:" into "😀"
+     
+     - Returns: Emoji string
+     */
+    public func emojiString() -> String {
+        return String.emojiStringFromString(self)
+    }
+    
+    /**
+        Converts emoji markdown to an emoji. For example, ":smile:" into "😀"
+     
+        - Parameter inputString: Emoji markdown string, such as ":smile:"
+        
```

Comment:
Is there a standard doc format that you're conforming to here? Wondering about the extra newlines.